src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1832,10 +1832,40 @@ namespace rpc {
       KV_MAP_SERIALIZABLE
     };
 
+    struct quorum_worker
+    {
+      std::string hash; // Public key of the worker service node
+      uint64_t uptime = 0; // Uptime of the worker service node
+      BEGIN_KV_SERIALIZE_MAP()
```

Comment:
I see that this PR has been updated for dev branch, can we move these serialization macros into the cpp file to follow suite with the other declarations here?